paywall/src/components/checkout/Checkout.tsx

```diff
@@ -36,15 +37,16 @@ export const Checkout = ({
         <p>{config.callToAction.default}</p>
       </Header>
       <CheckoutLocks>
-        {Object.values(locks).map(lock => {
+        {lockAddresses.map(lockAddress => {
+          const lock = locks[lockAddress]
```

Comment:
the key is lower-cased. The lock address is checksummed, and so case sensitive. This change ensures we map locks to their configuration names

Reply:
Do you think you could add your github comment into the code here? Just because this one is not so obvious and I could see someone coming back and "improving" this code in the future and inadvertently introducing a bug.